apps/server/src/modules/board/service/drawing-do-authorizable.service.ts

```diff
@@ -0,0 +1,92 @@
+import { AuthorizationLoaderService } from '@modules/authorization';
+import { forwardRef, Inject, Injectable } from '@nestjs/common';
+import {
+	AnyBoardDo,
+	BoardDoAuthorizable,
+	BoardExternalReferenceType,
+	BoardRoles,
+	ColumnBoard,
+	UserWithBoardRoles,
+} from '@shared/domain/domainobject';
+import { Course } from '@shared/domain/entity';
+import { EntityId } from '@shared/domain/types';
+import { CourseRepo } from '@shared/repo';
+import { DrawingDoRepo } from '../repo/drawing-do.repo';
+
+@Injectable()
+export class DrawingDoAuthorizableService implements AuthorizationLoaderService {
+	constructor(
+		@Inject(forwardRef(() => DrawingDoRepo)) private readonly boardDoRepo: DrawingDoRepo,
```

Comment:
it's confusing to refer to drawingBoardReo as boardDoRepo

Reply:
fixed